playbooks/engineer-workflow.md

```diff
@@ -82,6 +82,17 @@ _Assign_ another team member to the PR. All team members are encouraged to contr
 explicit responsibility to carefully review, consider any unanticipated impact, monitor discussion, and ultimately
 merge or close the PR. You may also want to add reviewers to request other developers for specific input.
 
+Who should you assign? You can ask yourself the following questions as a rubric:
+
+- Do I know who this PR affects and/or someone familiar with the codebase? Assign them.
+- Does the project have point persons listed in the README? Assign one of them.
+- Is the area targeted by your changes owned by a team in the [Project
+  list](https://github.com/artsy/potential/wiki/Project-List)? Assign someone
+  on that team.
+
+If none of these prompts yields a potential assignee, it's worth bringing up in
+Slack.
```

Comment:
GitHub suggests reviewers based on the history of the code that has been changed, so that _could_ also be an option to consider. I’ve definitely used this in the past to determine an assignee.

Reply:
That's a great suggestion! I'll add that to the rubric. 👍